src/features/hotspots/root/HotspotsView.tsx

```diff
@@ -282,7 +282,8 @@ const HotspotsView = ({
     }
 
     fetchHotspot()
-  }, [dispatch, handlePresentDetails, params])
```

Comment:
This used to work right? What changed?

Reply:
I think handlePresentDetail now has a dependency that causes it to change when you deep link

Reply:
which causes the useEffect to run again

Reply:
Ah ok